draft-ietf-quic-transport.md

```diff
@@ -1525,6 +1519,36 @@ initial_max_uni_streams (0x0008):
   equivalent to receiving a MAX_STREAM_ID containing 18 when received by a
   client or 19 when received by a server.
 
+initial_max_stream_data_bidi_local (0x0000):
+
+: The initial stream maximum data for bidirectional, local streams parameter
```

Comment:
The verbage we've used elsewhere is "[server|client]-initiated [bi|uni]directional," I think.  Obviously we can't do exactly that with these, but perhaps "locally-initiated bidirectional" and "peer-initiated bidirectional" would be closer?